locales/en-US/app.ftl

```diff
@@ -96,6 +127,15 @@ Home--additional-content-content = You can <strong>drag and drop</strong> a prof
 Home--compare-recordings-info = You can also compare recordings. <a>Open the comparing interface.</a>
 Home--recent-uploaded-recordings-title = Recent uploaded recordings
 
+## IdleSearchField
```

Comment:
What about adding some more comment explaining that this component is used for all search inputs in the application?